apps/site/playwright.config.ts

```diff
@@ -30,3 +30,17 @@ export default defineConfig({
     },
   ],
 });
+
+function getWebServerConfig(): Pick<Config, 'webServer'> {
+  if (process.env.PLAYWRIGHT_RUN_CLOUDFLARE_PREVIEW) {
+    return {
+      webServer: {
+        command: 'pnpm turbo run cloudflare:preview',
```

Comment:
This should be documented somehwere. If we ever updated our package.json this will silently break.

Maybe we can import package.json as a json file (import assert json) verify this entry is in the scripts, and then use it. Otherwise throw an exception.

Reply:
If you remove the script from package.json it would break; if the command doesnt exist or fails would also break, but at least with the check we can explicitly err saying hey missing script!

Reply:
I see, that's what I thought you were thinking, thanks for the clarification :slightly_smiling_face: